app/panel/containers/StatsContainer.js

```diff
@@ -22,7 +22,7 @@ import Stats from '../components/Stats';
  * @todo  We are not using ownProps, so we better not specify it explicitly,
  * in this case it won't be passed by React (see https://github.com/reactjs/react-redux/blob/master/docs/api.md).
  */
-const mapStateToProps = state => Object.assign({}, state.account);
+const mapStateToProps = state => Object.assign({}, state.account, {});
```

Comment:
Is that empty object at the end intended?

Reply:
Just a remnant of when I was adding theme props to this container before we changed to context. Deleted!